code/src/components/Form.js

```diff
@@ -0,0 +1,39 @@
+import React from "react";
+
+const Form = ({ newThought, setNewThought, formSubmit }) => {
+  return (
+    <div className="form-wrapper">
+      <form onSubmit={formSubmit}>
+        <label htmlFor="newThought">What's making you happy right now?</label>
+        <div>
+          <textarea
+            id="newThought"
+            type="text"
+            rows="5"
+            value={newThought}
+            onChange={(e) => setNewThought(e.target.value)}
+            placeholder="Happy thought here!"
+          />
+          <p className="letter-counter">
+            {140 - newThought.length} characters left
+          </p>
```

Comment:
This was a way simpler solution than the one I came up with! (I made a whole separate component for the character counter). Good job.